stacks/tcpconn.go

```diff
@@ -50,12 +51,12 @@ type TCPConn struct {
 	raddr, laddr net.TCPAddr
 }
 
-type TCPConnConfig struct {
+type TCPConConfig struct {
```

Comment:
Woah, revert back.

Reply:
Sorry - done (not sure how that happened)